src/test/java/BaseTest.java

```diff
@@ -1,11 +1,180 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.edge.EdgeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Parameters;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.UUID;
+
 
 public class BaseTest {
 
+    public static WebDriver driver = null;
+    public  String url = "https://qa.koel.app/";
+    public  WebDriverWait wait = null;
+    public  Actions actions;
+
+    private static final ThreadLocal<WebDriver> threadDriver = new ThreadLocal<>();
+
+    public WebDriver lambdaTest() throws MalformedURLException {
+        String Username = "Randy";
+        String authkey = "UPjcewrfPMqeXSaMBd4lo0aTfISKVyPnROLCNrZ5QowZoBzxWK";
+        String hub = "@hub.lambdatest.com";
+
+        DesiredCapabilities caps = new DesiredCapabilities();
+        caps.setCapability("platform", "windows 11");
+        caps.setCapability("browserName", "chrome");
+        caps.setCapability("version", "117");
+        caps.setCapability("resolution", "1024x768");
+        caps.setCapability("build", "TestNG with Java");
+        caps.setCapability("name", this.getClass().getName());
+        caps.setCapability("plugin", "git-testing");
+        return new remoteWebdriver(new URL("https://" +  username + ":" + authkey + hub), caps);
+    }
+    public WebDriver lambdaTest() throws MalformedURLException {
+        String hubURL = "https://lambdatest.com/wd/hub";
+
+        ChromeOptions browserOptions = new ChromeOptions();
+        browserOptions.setPlatformName("windows 11");
+        browserOptions.setBrowserVersion("117");
+        HashMap<String, Object> ltOptions = new HashMap<~>();
```

Comment:
Hello,
Your homework 24 has been checked, and Test script is failed! You got this problem, please correct it.
Best wishes from the HW team (Khanh, khanh@testpro.io)
![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/144403954/dc8f48ca-5905-4acb-bd22-c3868257c500)

Please check  systax HashMap and in the BaseTest have 2 methods lambdaTest